meqa/mqplan/dsl.go

```diff
@@ -1390,6 +1390,21 @@ func generateValue(valueType string, s mqswag.SchemaRef, prefix string) (interfa
 	return result, err
 }
 
+func generatePattern(format string) string {
+	switch format {
+	case "date-time":
+		return "^([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])[Tt]([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9]|60)(\\.[0-9]+)?(([Zz])|([\\+|\\-]([01][0-9]|2[0-3]):[0-5][0-9]))$" // https://gist.github.com/marcelotmelo/b67f58a08bee6c2468f8/
+	case "date":
+		return "^([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])$"
```

Comment:
We can `^([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])` keep this value at single place.